client/src/charts/NivoCharts.scss

```diff
@@ -7,4 +7,13 @@
   -webkit-box-shadow: none;
   color: $tertiaryColor !important;
   border-radius: $standardBorderRadius;
-}
\ No newline at end of file
+}
+
+.btn-table-view { 
+  background-color: $backgroundColor;
+  border: 2px solid $secondaryColor;
+  box-shadow: none;
+  -webkit-box-shadow: none;
+  color: $secondaryColor !important;
+  border-radius: $standardBorderRadius;
+} 
```

Comment:
Does this need a new class, or even a non-standard appearance? I think it should either be an icon or look the same as any other button.

Reply:
Yeah I'm still not 100% sure how this should look. I thought an icon would be fun (and relatively unobtrusive) but didn't want to put in the work (yet)